src/firefly/js/tables/ui/BasicTable.jsx

```diff
@@ -30,21 +28,18 @@ export class BasicTable extends React.Component {
             columnWidths: makeColWidth(props.columns, props.data, props.showUnits)
         };
 
-        this.onResize = this.onResize.bind(this);
+        this.onResize =  debounce((size) => {
+                if (size) {
+                    var widthPx = size.width;
+                    var heightPx = size.height;
+                    this.setState({widthPx, heightPx});
+                }
+            }, 100);
```

Comment:
Another comment i want to make is related to the design flow. When i open a different table, the tab currently shown is always the first tab. I don't know if this is the expected behavior but i was more in favor of getting automatically the tab switch to the recent table loaded. What do you think?